database/models/soundtracks.js

```diff
@@ -1,23 +1,23 @@
 const mongoose = require('mongoose');
 
 const soundSchema = new mongoose.Schema({
-  bundle_id: { type:Number, unique: true },
+  bundle_id: { type: Number, unique: true },
   bonus_info: [
     {
       title: String,
       composer_performer: String,
+      cover: String,
       tracklist: [
         {
           track_number: String,
           name: String,
           song: String
         }
-      ],
-      cover: String
+      ]
     }
   ]
 });
 
 const Soundtrack = mongoose.model('Soundtrack', soundSchema);
 
-module.exports = Soundtrack;
\ No newline at end of file
+module.exports = Soundtrack;
```

Comment:
Moved cover to appear before the tracklist array for readability and add a new line after the last line.